server/controller/module/sendMail.ts

```diff
@@ -0,0 +1,59 @@
+import { createTransport } from "nodemailer";
+import * as dotenv from "dotenv";
+import path from "path";
+
+//The below config setup was unable to be resolved. Need to connect and see on this
+
+const config = dotenv.config({
```

Comment:
this function can be replaced by [loadConfigs](https://github.com/siddharthrajaraja/ChatApp/blob/d0ed80799571be077031a96ab4e3190e65843e2f/server/loadConfigs.ts#L12) where we simply need to call these properties from load configs